src/main/java/com/adobe/target/edge/client/service/DefaultTargetService.java

```diff
@@ -53,8 +55,22 @@ public DefaultTargetService(ClientConfig clientConfig, TelemetryService telemetr
     } else {
       this.targetHttpClient = targetHttpClient;
     }
+
+    Properties defaultProps = new Properties();
+    try {
+      InputStream in = getClass().getResourceAsStream("/gradle.properties");
+      BufferedReader reader = new BufferedReader(new InputStreamReader(in));
+      defaultProps.load(reader);
+      in.close();
+    } catch (IOException e) {
+      System.out.println("Unable to determine Target SDK version");
```

Comment:
Let's use logger instead:
```suggestion
      logger.warn("Unable to load default SDK properties");
```

Reply:
why the change the message from "Unable to determine Target SDK version" to "Unable to load default SDK properties"? Wouldn't we want to be precise with what the issue is? Or do you foresee us incorporating more properties in the future into the `loadDefaultProperties` method?

Reply:
It's just more accurate and helpful to know that the error occurred when loading the properties, not when reading the version from loaded properties